ansible/roles/ml-analytics-service/defaults/main.yml

```diff
@@ -7,11 +7,11 @@ config_path: "/opt/sparkjobs/ml-analytics-service"
 SYSTEMDDIR: "/lib/systemd/system"
 ml_analytics_version: "release-4.1.0"
 ml_survey_cloud_storage: "AZURE"
-ml_analytics_survey_app_name: "{{ ml_cloud_config.CLOUD_STORAGE }}"
+ml_analytics_survey_app_name: "sunbirdsurvey"
```

Comment:
make it as a variable and give default value as sunbirdapp.
`"{{ ml_analytics_survey_app_name|default('sunbirdsurvey') }}"`